src/custom-element.js

```diff
@@ -304,6 +308,9 @@ export function createAmpElementProto(win, name, implementationClass) {
     /** @private {!SizeList|null|undefined} */
     this.sizeList_;
 
+    /** @private {!AspectList_|null|undefined} */
+    this.aspectList_;
```

Comment:
Looks like it should be `heightsList_` here as well. Also, the type, I assume, should be `SizeList`?